highlight/highlight
# Support saved segments (backend)

## Summary
Create the necessary tables and graphql mutations/queries needed to support saved searches for the traces and logs table.

## How did you test this change?
- Check code is valid.
- Click test in https://github.com/highlight/highlight/pull/7408

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
N/A
